runtime/turing/src/lib.rs

```diff
@@ -573,6 +573,8 @@ impl cumulus_pallet_xcmp_queue::Config for Runtime {
 	type ChannelInfo = ParachainSystem;
 	type VersionWrapper = ();
 	type ExecuteOverweightOrigin = EnsureRoot<AccountId>;
+	type ControllerOrigin = EnsureRoot<AccountId>;
```

Comment:
this is the controller account for sudo?

Reply:
yes